src/xo-app/backup/new/index.js

```diff
@@ -5,26 +5,40 @@ import Component from 'base-component'
 import GenericInput from 'json-schema-input'
 import getEventValue from 'get-event-value'
 import Icon from 'icon'
+import Link from 'link'
 import moment from 'moment-timezone'
+import propTypes from 'prop-types-decorator'
```

Comment:
Let's stop using this module, use `prop-types` directly :)

https://github.com/vatesfr/xo-web/commit/d7f8d12d888718be09788d638bbb30b0debc679b